src/app/components/MediaLoader/index.tsx

```diff
@@ -241,7 +241,8 @@ const MediaLoader = ({ blocks, className, embedded, uniqueId }: Props) => {
 
   const hasPlaceholder = Boolean(showPlaceholder && placeholderSrc);
 
-  const showPortraitTitle = orientation === 'portrait' && !embedded;
+  const showPortraitTitle =
+    orientation === 'portrait' && pageType !== MEDIA_ARTICLE_PAGE && !embedded;
```

Comment:
I am wondering if this logic (and the bit that renders the portrait title) should be moved out of here, and into the Article page. I don't think this title should be part of the media loader component, as I believe it should only be for the media player itself.

Reply:
I agree. We could have a prop / options that are passed into MediaLoader, but are actually determined/set outside of the component by another component like the MediaArticlePage.

Reply:
I actually mean that we should remove not only this boolean but also the rendering of the portrait title (lines 255-258). The title is only required on Article pages, so I think it should be moved there instead.

Reply:
Ah true. Makes sense 👍